CMSgov/design-system
# [WNMGDS-922] Update stackable table

## Summary
Update stackable tables:
- disable console warning message of 'the header prop is required' and 'the stackedTitle prop is required' for a column of row headers (issue: #1043)
- disable console warning message of 'the id prop is required' for a column header without any children value

Re-instate the use of Context for Table component now that consuming applications have upgraded to newer version of React.

## How to test
- Check that http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-922/stackable-table-updates remains unchanged when compared to design.cms.gov

- Update the stackable table component example `Table.example.jsx`:
   - change line 52:  `<TableCell />`
   - change line 59 and 71:   `<TableCell component="th" scope="row">`

- Run `yarn start` on master and see console warnings
- Run `yarn start` on this branch and see there are no errors
